geonode_mapstore_client/static/mapstore/configs/localConfig.json

```diff
@@ -778,6 +782,13 @@
                                 }
                             ]
                         },
+                        {
+                            "type": "locations",
+                            "id": "locations",
+                            "disableIf": "{['dataset', 'map'].includes(context.get(state('gnResourceData'), 'resource_type'))}",
```

Comment:
The locations panel should be available also for map and dataset without allowing the editing

Reply:
Yes. Not needed when map should be visible for all resource